package.json

```
@@ -4,10 +4,10 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
-    "start": "node ./src/bin/www",
+    "start": "set DEBUG=dev && node ./src/bin/www",
     "devstart": "babel-watch ./src/index.js",
```

Please let's start our server with the `/src/bin/www` file even locally. 
```suggestion
    "devstart": "babel-watch ./src/bin/www",
```

---

ok